schemas/io-package.json

```diff
@@ -115,7 +115,86 @@
+                                "BSD-3-CLAUSE",
+                                "bsd-3-clause-clear",
+                                "BSD-3-CLAUSE-CLEAR",
+                                "cc",
+                                "CC",
+                                "cc0-1.0",
+                                "CC0-1.0",
+                                "cc-by-4.0",
+                                "CC-BY-4.0",
+                                "cc-by-sa-4.0",
+                                "CC-BY-SA-4.0",
+                                "wtfpl",
+                                "WTFPL",
+                                "ecl-2.0",
+                                "ECL-2.0",
+                                "epl-1.0",
+                                "EPL-1.0",
+                                "epl-2.0",
+                                "EPL-2.0",
+                                "eupl-1.1",
+                                "EUPL-1.1",
+                                "agpl-3.0",
+                                "AGPL-3.0",
+                                "gpl",
+                                "GPL",
+                                "gpl-2.0",
+                                "GPL-2.0",
+                                "gpl-3.0",
+                                "GPL-3.0",
+                                "lgpl",
+                                "LGPL",
+                                "lgpl-2.1",
+                                "LGPL-2.1",
+                                "lgpl-3.0",
+                                "LGPL-3.0",
+                                "isc",
+                                "ISC",
+                                "lppl-1.3c",
+                                "LPPL-1.3C",
+                                "ms-pl",
+                                "MS-PL",
+                                "mit",
+                                "MIT",
+                                "mpl-2.0",
+                                "MPL-2.0",
+                                "osl-3.0",
+                                "OSL-3.0",
+                                "postgresql",
+                                "POSTGRESQL",
+                                "ofl-1.1",
+                                "OFL-1.1",
+                                "ncsa",
+                                "NCSA",
+                                "unlicense",
+                                "UNLICENSE",
+                                "zlib",
+                                "ZLIB"
+                            ]
+                        },
+                        {
+                            "type": "string",
+                            "pattern": "[a|A][r|R][t|T][i|I][s|S][t|T][i|I][c|C]-2.0|[a|A][p|P][a|A][c|C][h|H][e|E]-2.0|[p|P][o|O][s|S][t|T][g|G][r|R][e|E][s|S][q|Q][l|L]|[u|U][n|N][l|L][i|I][c|C][e|E][n|N][s|S][e|E]"
+                            "$comment": "Some are case insensitive, we have to handle them separately"
+                        }
```

Comment:
I would just add another enum value `Apache-2.0`, `PostgreSQL` and `Unlicense` instead.